request/request.go

```diff
@@ -0,0 +1,61 @@
+package request
+
+import (
+	"encoding/binary"
+	"encoding/json"
+	"io"
+	"os"
+
+	"github.com/browserpass/browserpass-native/errors"
+	"github.com/browserpass/browserpass-native/response"
+	log "github.com/sirupsen/logrus"
+)
+
+type request struct {
+	Action   string      `json:"action"`
+	Settings interface{} `json:"settings"`
+}
+
+// Process handles browser request
+func Process() {
+	requestLength := parseRequestLength()
+	request := parseRequest(requestLength)
+
+	switch request.Action {
+	case "configure":
+		break
+	case "list":
+		break
+	case "fetch":
+		break
+	default:
+		log.Errorf("Received a browser request with an unknown action: %+v", request)
```

Comment:
Missing call to `response.SendError()` here - are you intending to add this later? The extension needs to know if it sends an action that the host app doesn't support (even though it should never issue such a request).

Reply:
Oh, great catch!